src/components/common/ClientIntakeForm/index.js

```diff
@@ -127,5 +133,3 @@ export const FamilyIntake = () => {
     </div>
   );
 };
```

Comment:
Nice job changing this. This should help avoid confusion

Reply:
Thank you!